apps/docs/src/app/core/component-docs/select/examples/select-adding-example/select-adding-example.component.html

```diff
@@ -3,7 +3,7 @@
     <button fd-button (click)="removeOption()">Remove Option</button>
 </div>
 
-<fd-select placeholder="Select an option" [(value)]="selectedValue" [closeOnOutsideClick]="false">
-    <fd-option *ngFor="let option of options" [value]="option">{{option}}</fd-option>
+<fd-select placeholder="Select an option" [(value)]="selectedValue">
```

Comment:
@JKMarkowski This is one of the thing why this is braking change. this close on click was feature of popover as I saw. and Select handles this naturally. Then it would not be really a select. I wanted to do also little cleanup, fresh start.